.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: Integration Test
+
+on:
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - '.github/workflows/integration-test.yml'
+      - 'docker/**'
+
+  push:
+    branches:
+      - develop
+    paths:
+      - '.github/workflows/integration-test.yml'
+      - 'docker/**'
+
+  release:
+    types: [published]
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout dea-sandbox
+        uses: actions/checkout@v3
+        with:
+          path: dea-sandbox
+
+      - name: git checkout dea-notebooks
+        uses: actions/checkout@v3
+        with:
+          repository: GeoscienceAustralia/dea-notebooks
+          path: dea-notebooks
+          ref: clean_test_workflow
```

Comment:
We should use `stable` I think - this is the branch that gets used to load DEA Notebooks into the Sandbox, and render the official user guide

Reply:
As commented in the PR note, there will be change on the branch after the pr in `dea-notebooks` is merged. `stable` doesn't have the bit this pr required yet.